projects/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart.component.ts

```diff
@@ -65,4 +67,7 @@ export class FhirChartComponent implements OnInit {
       this.options = config.options;
     });
   }
+  ngAfterContentChecked(): void {
+    this.changeDetector.detectChanges();
+  }
```

Comment:
why is this needed?

Reply:
I was getting this error when I am passing canvas reference to a child, I search for this error and we got a solution we add this because the reference is  undefined, and then it will be defined, I also manually check that but then zoom is not working 
![image](https://user-images.githubusercontent.com/117890382/211491584-9a3a2d19-aa97-4037-8411-adc0fe7ba5de.png)

Reply:
Running an extra change detection cycle doesn't seem like the best solution to `ExpressionChangedAfterItHasBeenCheckedError`. I'm guessing this has something to do with passing the component reference around. Maybe we could use a service to share the chart object instead?